### ISSUE TITLE
models: file exports do not include checksums or size

### ISSUE THREAD

**Issue by GeoffFroh** [densho/ddr-defs#1]
`ddrexport` relies on the models in `ddr-defs` to determine which attributes end up in the exported data. Currently, `file` exports do not include `sha1`, `sha256`, `md5`, or `size`, which are needed by the archivists.

See, e.g.: `https://github.com/densho/ddr-defs/blob/master/repo_models/files.py#L118`

**Comment by gjost** [comment]
Looks like those fields are excluded when exporting to CSV:
`repo_models.files.FIELDS_CSV_EXCLUDED`

The question is *why* I excluded them.  I'll check and see if there's any problem with including them and reimporting.

**Comment by gjost** [comment]
Looks like it's not quite so simple...
```
repo_models.files.REQUIRED_FIELDS_EXCEPTIONS
repo_models.files.FIELDS[FIELD]['csv'][export/import']
repo_models.files.FIELDS_CSV_EXCLUDED
```

**Comment by GeoffFroh** [comment]
fixed in 2b481bc9afeb4e873b56f19c053a3d41b3966b95

Ready for merge into `master` and deployed
